applications/RomApplication/python_scripts/RSVDT_Library.py

```diff
@@ -0,0 +1,460 @@
+### RANDOMIZED TRUNCATED SINGULAR VALUE DECOMPOSITION FUNCTIONS  ###

```

Comment:
I think we should discuss about this. It would be nice to have this capabilities either in the core or in the external libraries so they are available for every one.

Besides I think that these utilities should be implemented in c++ for the sake of performance.

Reply:
@rubenzorrilla i disagree with you on this: this function is intended for use in python, in a preprocessing step.
It is NOT used during the calculation and the performance impact of using python should be negligible as all the heavy lifting is done in lapack.

As an aside, doing it in python we take advantage of scipy.linalg in wrapping lapack for the computations, and i REALLY do not want to go through the pain of doing that wrapping myself within kratos.

Reply:
I think that now I understand the usage of this and it would thus make sense to keep it in Python (my aplogies, I didn't realise that the linalg is used).


However I insist on the format (make it in a Kratos fashion) and the location. I think that it is nice to have these features available in the core.

Reply:
After a brief discussion with @RiccardoRossi we agreed on keeping this outside the Kratos Core to ease future developments. As soon as it is production ready, it will be placed in there.